superbench/analyzer/data_diagnosis.py

```diff
@@ -279,6 +277,7 @@ def output_diagnosis_in_excel(self, raw_data_df, data_not_accept_df, output_path
             rules (dict): the rules of DataDiagnosis
         """
         try:
+            data_not_accept_df = data_not_accept_df.convert_dtypes(convert_integer=True).astype('object').fillna('')
```

Comment:
yes, it's default, but why I can't explictly write here though it's default? any concerns on that?

Reply:
> DataFrame.convert_dtypes(infer_objects=True, convert_string=True, convert_integer=True, convert_boolean=True, convert_floating=True)

there're five default parameters, why do you only write one??

Reply:
because 'convert_integer' is the most necessary one, if others's default change to False (if api change), it's ok for me

Reply:
that sounds strange, so if convert_floating=False, you will have 1 (int) and "1.1" (string) in the results and it's still ok?

Reply:
no, I said it‘s ok because before converting, data_not_accept_df are all float64 for the benchmark metrics. But we don't need to argue all the time, what's you prefer, convert_dtypes() or convert_dtypes(convert_objects=True, convert_string=True, convert_integer=True, convert_boolean=True, convert_floating=True)